approximate_moran.py

```diff
@@ -28,10 +35,9 @@ class ApproximateMoranProcess(axl.MoranProcess):
     Instead of playing the matches, the result is sampled
     from a dictionary of play tuples to distribution of match outcomes
     """
-    def __init__(self, players, cached_outcomes=None, turns=0,
-                 noise=0, mutation_rate=0.):
+    def __init__(self, players, cached_outcomes, mutation_rate=0.):
```

Comment:
Not sure why we're fixing the parameters here -- we'll want to set `noise != 0` eventually.